service-base/src/main/java/org/eclipse/hono/service/tenant/TenantHttpEndpoint.java

```diff
@@ -128,32 +125,35 @@ private void addTenant(final RoutingContext ctx) {
 
         final String location = String.format("/%s/%s", TenantConstants.TENANT_ENDPOINT, tenantId);
 
-        doTenantHttpRequest(ctx, tenantId, StandardAction.ACTION_ADD, getHttpStatusPredicate(HTTP_CREATED),
-                getLocationHeaderHandler(location));
+        doTenantHttpRequest(ctx, tenantId, RequestResponseApiConstants.StandardAction.ACTION_ADD,
+                status -> status == HttpURLConnection.HTTP_CREATED,
+                response ->  response.putHeader(HttpHeaders.LOCATION, location)
+        );
     }
 
     private void getTenant(final RoutingContext ctx) {
 
         final String tenantId = getTenantIdFromContext(ctx);
 
-        doTenantHttpRequest(ctx, tenantId, StandardAction.ACTION_GET, getHttpStatusPredicate(HTTP_OK), null);
+        doTenantHttpRequest(ctx, tenantId, RequestResponseApiConstants.StandardAction.ACTION_GET,
+                status -> status == HttpURLConnection.HTTP_OK, null);
     }
 
     private void updateTenant(final RoutingContext ctx) {
 
         final String tenantId = getTenantIdFromContext(ctx);
 
-        doTenantHttpRequest(ctx, tenantId, StandardAction.ACTION_UPDATE,null, null);
+        doTenantHttpRequest(ctx, tenantId, RequestResponseApiConstants.StandardAction.ACTION_UPDATE,null, null);
```

Comment:
I think we should refer to the actions via the `TenantConstants` class (this is where they will end up anyway FMPOV)

Reply:
👍